presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,28 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
```

Comment:
Or look into libraries, maybe there is one ready for use. Just like the `HtttpMethod` enum.